File: POO/Python/Prueba Clases/Persona.py
class Persona:
    cedula = 0
    nombre = ""
    apellido = ""

    def __init__(self,_cedula,_nombre,_apellido):
        self.cedula = _cedula
        self.nombre = _nombre
        self.apellido = _apellido
    
    def toString(self):
        return str(self.cedula),str(self.nombre),str(self.apellido)

File: POO/C++/EjercicioSumativoUn9/unidad9-10/Persona.cpp
#include "Persona.h"

Persona::Persona()
{
    cedula = 0;
    nombre = "";
    apellido = "";
}

Persona::Persona(int c, string n, string a)
{
    //constructor con parametros
    cedula = c;
    nombre = n;
    apellido = a;
}

Persona::~Persona() {
    //destructor
}

string Persona::toString(){
    ostringstream a;
    a <<"Cedula: "<<cedula<<" Nombre: "<<nombre<<" Apellido: "<<apellido;
    return a.str();
}

void Persona::comer(string comida,float cantidad){
    cout<<"\n\tComiendo: "<<comida<< " cant: "<<cantidad<<endl;
}
